cluster-usermin/cluster-usermin-lib.pl

```diff
@@ -113,7 +113,7 @@ sub all_themes
 {
 local %done;
 return sort { $a->{'desc'} cmp $b->{'desc'} }
-	grep { !$done{$_->{'dir'}}++ }
+	grep { ! -l "$root_directory/$_->{'dir'}" && !$done{$_->{'dir'}}++ }
```

Comment:
Is this function used anywhere other than to display a UI for the user? If so, the filtering should be done at the UI level, because a user might still have a linked theme selected.

Reply:
Yes, correct. Unrolled. 
https://github.com/webmin/webmin/pull/1453/commits/38deb9c1328e9a3be6f775811aa52a62267b5868